base/core/src/main/java/org/openscience/cdk/config/AtomTypeFactory.java

```diff
@@ -280,8 +281,7 @@ public IAtomType getAtomType(String identifier) throws NoSuchAtomTypeException {
         logger.debug("Request for atomtype for symbol ", symbol);
         List<IAtomType> atomList = new ArrayList<IAtomType>();
         for (IAtomType atomType : atomTypes.values()) {
-            // logger.debug("  does symbol match for: ", atomType);
-            if (atomType.getSymbol().equals(symbol)) {
+            if (Objects.equals(atomType.getSymbol(), symbol)) {
```

Comment:
Just wondering (and learning), what's the advantage of this? That both can be null? Or is there more?